flyway/scxa/migrations/V18__anndata_import_changes.sql

```diff
@@ -0,0 +1,50 @@
+-- Create a table holding info on a dimension reduction, most of which was previously in scxa_coords 
+
+CREATE TABLE scxa_dimension_reduction
+(
+    id SERIAL not null PRIMARY KEY,
+    experiment_accession VARCHAR(255) NOT NULL,
+    method VARCHAR(255) NOT NULL,
+    parameterisation JSONB DEFAULT NULL,
+    priority SMALLINT DEFAULT 0 NOT NULL,
```

Comment:
@pinin4fjords can you clarify if this is intended to be the order of visualization. Is it highest number or lowest number the default? or we are missing the mentioned default flag?

Reply:
This is instead of default. From the PR description:

```
New priority field on scxa_dimension_reduction. This was originally discussed as a boolean 'default' field, but I think a numeric priority is more flexible. The UI would sort the dimension reductions for an experiment in descending order and pick the first. Initially we'll only set one DR (our default) as non-zero, but in future this will allow more detailed prioritisation.
```